src/plot.c

```diff
@@ -29,6 +25,10 @@ plot_init(struct plot *plot)
 
 	plot->follow_rate = 100;
 	plot->average = 1;
+
+	plot->datasets = 0;
```

Comment:
why re-initialize plot->datasets to 0?

Reply:
`plot_init` did not initialize datasets before this fixup commit.

Reply:
true, it was the callers (fulfilled) responsibility :^).  But I think this is better